code/lib/cli/src/automigrate/index.ts

```diff
@@ -211,10 +212,19 @@ export async function runFixes({
     let result;
 
     try {
+      const { mainConfig, previewConfigPath } = await getStorybookData({
+        configDir,
+        packageManager,
+      });
+
       result = await f.check({
         packageManager,
         configDir,
         rendererPackage,
+        mainConfig,
+        storybookVersion,
```

Comment:
given that this is not a coerced version, it will break in places that use it with `semver.gte` etc
```suggestion
        storybookVersion: sbVersionCoerced,
```

Reply:
@yannbf Can you show me some examples where you think this change breaks the current behaviour?